keyboards/melgeek/z70ultra_v1/readme.md

```diff
@@ -0,0 +1,27 @@
+# Z70 Ultra v1
+
+![z70ultra](https://drive.google.com/file/d/1m8Qra8L7Nq4I-ewq1_GkKi9j0LVPN7_l/view?usp=sharing)
```

Comment:
1. url is not image file
2. can image be re-hosted in Imgur

Reply:
yes